src/main/webapp/css/one-page-wonder.css

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Start Bootstrap - One Page Wonder v5.0.8 (https://startbootstrap.com/template-overviews/one-page-wonder)
+ * Copyright 2013-2020 Start Bootstrap
+ * Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-one-page-wonder/blob/master/LICENSE)
+ */
+
```

Comment:
Consider structuring the CSS in sections based on the page.
Ex.

> /* Header section */
> .btn{ }
> header { .. }

Reply:
I actually deleted this file because I realized it was unnecessary :)